packages/components/src/Card/Card.css

```diff
@@ -42,7 +42,6 @@
 .clickable {
   --card--border: none;
```

Comment:
I think we just need to delete this line (and move it to the elevation styles?)

Reply:
Ah good call, sorry I should have caught this based on the Figma. Thank you for pointing it out